website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/example-customizations.mdx

```diff
@@ -7,190 +7,145 @@ description: Example customizations for the ServiceNow Service Catalog Integrati
-   1. Once the Flow is properly configured, click "Save" at the top to save the
-      changes.
-   1. Click "Activate" to enable your Flow and make it available for use.
+   1. Click **Save**.
+   1. Click **Activate**.
 
-### Set the Flow for your Catalog Item
+## Set the Flow for your Catalog Item
 
-1. Navigate back to the Catalog by clicking on "All" and then go to "Service
-   Catalog > Catalogs" followed by "Terraform Catalog".
-1. Locate your custom Catalog Item and click on the three dots icon at the top
-   of the item. From the dropdown menu, select "Configure item".
-1. In the configuration settings, click on the "Process Engine" tab.
-1. In the "Flow" field, search for the Flow you recently created. Once you find
-   it, click on it to select it and then click the "Update" button to save the
-   changes.
-   ![Screenshot: ServiceNow Update Process Engine](/img/docs/servicenow-catalog-update-process-engine.png "Screenshot of updating Process Engine for the Catalog Item")
+1. Navigate back to the Catalog by clicking on **All** and then go to **Service
+   Catalog > Catalogs > Terraform Catalog**.
+1. Locate your custom Catalog Item and click **...** at the top
+   of the item. From the dropdown menu, select **Configure item**.
+1. In the configuration settings, click the **Process Engine** tab.
+1. In the **Flow** field, search for the Flow you recently created. Click 
+   the Flow then click the **Update**.
 
-### Test the Catalog Item
+## Test the Catalog Item
 
-The new item should be available in the Terraform Service Catalog. To make the
+The new item is now available in the Terraform Service Catalog. To make the
 new item accessible to your end users via the Service Portal, follow these
 steps:
-1. Navigate to the configuration page of the item you want to make available as
-   described in the previous section.
-1. Locate the "Catalogs" field on the configuration page and click on the lock
+1. Navigate to the configuration page of the item you want to make available.
+1. Locate the **Catalogs** field on the configuration page and click the lock
    icon next to it.
 1. In the search bar, type "Service Catalog" and select it from the search
    results. Add "Service Catalog" to the list of catalogs associated with the
-   item. Click on the lock icon again to lock the changes.
+   item. Click the lock icon again to lock the changes.
    ![Screenshot: ServiceNow Enable Service Portal](/img/docs/servicenow-catalog-service-portal.png "Screenshot of adding the Catalog Item to the Service Portal")
-1. Finally, click the "Update" button at the top of the page to save the
-modifications. After completing these steps, your end users will be able to
-access the new item through the Service Portal of your ServiceNow instance. The
-Service Portal can be accessed by navigating to All > Service Portal Home.
```

Comment:
Please keep the path to the Service Portal. Many customers really don't know what it is or how to get there.

Reply:
`The Service Portal can be accessed by navigating to All > Service Portal Home.` <<< there's a lot of confusion about this even among our own SMEs. I get questions about this in Slack every month. Please bring back this sentence.